docs/docs/development/backend.mdx

```diff
@@ -103,10 +103,7 @@ infrahub server start --debug
 
 The `debug` flag allows the server to be reloaded when a change is detected in the source code.
 
-Note that this will only make the backend service usable, the frontend will not be available. Only the following URLs should work:
-
-* http://localhost:8000/api/docs/
-* http://localhost:8000/graphql/
+Note that this will only make the backend service usable, the frontend will not be available. Only http://localhost:8000/api/docs should work.
```

Comment:
Thanks for the update!

How about mentioning that `/api/docs/` is for the Swagger documentation? With the current wording, it could be misunderstood as if only one endpoint is available.

Also do you feel like it could be useful to clarify that `http://localhost:8000/graphql/` (GraphQL documentation/sandbox) will be accessible only through the frontend application.

Reply:
Yes it makes sense